docs/source/checks/tabular/data_integrity/plot_feature_feature_correlaion.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+Feature Feature Correlation
+***************************
+This notebooks provides an overview for using and understanding the feature-feature correlation check.
+
+This check computes the pairwise correlations between the features, potentially spotting pairs of features that are
+highly correlated.
+
+**Structure:**
+
+* `How are The Correlations Calculated? <#how-are-the-correlations-calculated>`__
+* `Load Data <#load-data>`__
+* `Run the Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
+
+How are The Correlations Calculated?
+============================
+This check works with 2 types of features: categorical and numerical, and used a different method to calculate the
+correlation for each combination of feature types.
+
+1. numerical-numerical: Pearson's correlation coefficient
+2. numerical-categorical: Correlation ratio
+3. categorical-categorical: Symmetric Theil's U statistic
```

Comment:
Add links to articles (Wikipedia is fine) that explains all of those, as hyperlinks